staging-service/src/main/java/com/github/cafdataprocessing/services/staging/StagingController.java

```diff
@@ -140,19 +140,19 @@ public ResponseEntity<BatchStatusResponse> getBatchStatus(final String X_TENANT_
             final BatchStatusResponse statusResponse = batchDao.getBatchStatus(new TenantId(X_TENANT_ID), new BatchId(batchId));
             return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(statusResponse);
         } catch (final InvalidTenantIdException ex) {
-            LOGGER.error("Invalid X-TENANT-ID", ex);
+            LOGGER.warn("Invalid X-TENANT-ID {}", X_TENANT_ID);
             throw new WebMvcHandledRuntimeException(HttpStatus.BAD_REQUEST, ex.getMessage());
         } catch (final InvalidBatchIdException ex) {
-            LOGGER.error("Invalid Batch Id", ex);
+            LOGGER.warn("Invalid Batch Id {}", batchId);
             throw new WebMvcHandledRuntimeException(HttpStatus.BAD_REQUEST, ex.getMessage());
         } catch (final BatchNotFoundException ex) {
-            LOGGER.error("Batch not found", ex);
+            LOGGER.warn("Batch {} not found", batchId);
             throw new WebMvcHandledRuntimeException(HttpStatus.NOT_FOUND, ex.getMessage());
         } catch (final StagingException ex) {
-            LOGGER.error("Internal server error", ex);
+            LOGGER.warn("Internal server error: {}", ex.getMessage());
             throw new WebMvcHandledRuntimeException(HttpStatus.INTERNAL_SERVER_ERROR, ex.getMessage());
         } catch (final InterruptedException ex) {
-            LOGGER.error("Service Unavailable", ex);
+            LOGGER.warn("Service Unavailable: {}", ex.getMessage());
```

Comment:
This looks like a case were we should pass ex?

Reply:
addressed.